core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -30,6 +30,7 @@
 import org.apache.thrift.TException;
 import org.apache.thrift.protocol.TProtocol;
 import org.apache.thrift.transport.TTransportException;
+import org.checkerframework.checker.nullness.qual.Nullable;
```

Comment:
Yes, it's possible to use ```javax.annotation.Nullable``` but with some limitations:

"The JSR 305 annotations have a nicer package name (javax.annotation) than the ones from the Checker Framework, but bear in mind that they are less powerful. Only the Checker Framework annotations can be applied to the use of a type. The following, ‘a non-null List of nullable Strings’ cannot be expressed with the JSR 305 annotations."

```private @NonNull List<@Nullable String> list;```

https://github.com/glts/safer-spring-petclinic/wiki/Setup

Reply:
Ok, since `org.checkerframework.checker.nullness.qual.Nullable` is more powerful I'm happy sticking with it. If we ever want to switch away, we can always do find/replace and remove the `Nullable`s that apply to types